src/main.rs

```diff
@@ -0,0 +1,55 @@
+use crate::commands::generate_buildpack_matrix::command::GenerateBuildpackMatrixArgs;
```

Comment:
Could you enable the usual lints additional lints we use for our Rust repos? :-)

```
#![warn(clippy::pedantic)]
#![warn(unused_crate_dependencies)]
```

Reply:
Filed #36